internal/cloud/atlas/groups_test.go

```diff
@@ -49,6 +51,28 @@ func TestAtlasGroups(t *testing.T) {
 	})
 }
 
+func TestFetchGroups(t *testing.T) {
+	u.SkipUnlessAtlasServerRunning(t)
+
+	t.Run("should fetch all groups", func(t *testing.T) {
+		for _, tc := range []struct {
+			numGroups int
+		}{
+			{numGroups: 1},
+			{numGroups: 3},
+			{numGroups: 5},
+		} {
+			t.Run(fmt.Sprintf("when num items allowed per page is %d", tc.numGroups), func(t *testing.T) {
+				client := newAuthClient(t)
+				options := api.RequestOptions{Query: map[string]string{"itemsPerPage": fmt.Sprint(tc.numGroups)}}
+				groups, err := client.FetchGroups(u.AtlasServerURL()+atlas.GroupsPath, options)
```

Comment:
why does this need to use `FetchGroups` instead of just `Groups`?

Reply:
specifically for testing, we pass in the `itemsPerPage` as an option (`Groups` doesn't take in any params).
setting `itemsPerPage = 1` in the query param to forces the fetch for more groups via links. (otherwise, without the param, all groups would be returned in the first request)